server/knex/migrations/20181029110454_create_table_jobs_and_related_tables.js

```diff
@@ -53,7 +53,7 @@ exports.up = (knex, Promise) => (async () => {
     await knex.schema.createTable(JOB_RUNS, table => {
         table.increments('id').primary();
         table.integer('job').unsigned().notNullable().references('jobs.id').onDelete('CASCADE').index();
-        table.timestamp('started_at').nullable().defaultTo(null);
+        table.timestamp('started_at').nullable().defaultTo(knex.fn.now());
```

Comment:
This should be fine. The problem was that MariaDB does not allow two timestamp attributes set the CURRENT_TIMESTAMP. 

However, it is still a bit weird that when a job is created, it get the `started_at` set to current timestamp even though the job may not be actually started. As far as I understand, creating it does not automatically start it. Or?

Reply:
And what happens when you store Date (in UTC) instead of a string?